test/export/test_torchbind.py

```diff
@@ -392,6 +403,52 @@ def forward(self, arg0_1, attr, arg1_1):
     return (getitem_3, add_1)""",  # noqa: B950
         )
 
+    def test_make_fx_tensor_queue_methods(self):
```

Comment:
IMO you should combine #122622 and #122623; there's no end-to-end example in 122622

Reply:
Do you mind if keep them as they're now? Because it's kind of tedious to merge them...

Reply:
Yes, that's fine.

Also, it shouldn't be hard to merge them, right? you can `git rebase -i HEAD~10`, then squash the commits together, then run `ghstack`.

Reply:
yeah, didn't use the squash command before hahaha